src/lavinmq/http/controller.cr

```diff
@@ -67,6 +67,8 @@ module LavinMQ
                 sorted_items.sort_by! { |i| dig(i, sort_by).as(Number) }
               when String
                 sorted_items.sort_by! { |i| dig(i, sort_by).as(String).downcase }
+              when QueueState
+                sorted_items.sort_by! { |i| dig(i, sort_by).as(QueueState).to_s.downcase }
```

Comment:
Is it necessary to convert to string here (and downcase)? Enums are comparable so shouldn't be needed?